server/src/controllers/eventController.ts

```diff
@@ -888,35 +873,30 @@ eventController.use(checkScopes);
  */
 eventController.post('/reader-intent', async (req: Request, res: Response) => {
   const {ticketCartItems} = req.body;
-  let paymentIntentID = '';
-  let clientSecret = '';
 
   try {
     if (!ticketCartItems.length) {
       return res.status(400).json({error: 'Cart is empty'});
     }
 
     const {
-      ticketCartRows,
-      orderTicketItems,
       ticketTotal,
       feeTotal,
-      eventInstanceQueries,
     } = await getTicketItems(ticketCartItems, prisma);
 
-    if (ticketTotal > 0) {
-      const {id, secret} = await createStripePaymentIntent((ticketTotal + feeTotal) * 100);
-      paymentIntentID = id;
-      clientSecret = secret;
+    if (ticketTotal + feeTotal < .50) {
+      return res.status(400).json({error: 'Reader checkout can not be used for an order below .50 USD'});
     }
-    res.json({id: paymentIntentID, secret: clientSecret});
+
+    const response = await createStripePaymentIntent((ticketTotal + feeTotal) * 100);
+    return res.json(response);
   } catch (error) {
     console.error(error);
     if (error instanceof InvalidInputError) {
       res.status(error.code).json(error.message);
       return;
     }
-    res.status(500).json(error);
+    res.status(500).json('Internal Server Error');
```

Comment:
Good catch, I've noticed we occasionally have an unsanitized error pop up on the client side.